src/librustc_metadata/native_libs.rs

```diff
@@ -169,6 +170,14 @@ impl Collector<'tcx> {
                                            GateIssue::Language,
                                            "kind=\"static-nobundle\" is feature gated");
         }
+        if lib.kind == cstore::NativeRawDylib &&
+           !self.tcx.features().raw_dylib {
+            feature_gate::emit_feature_err(&self.tcx.sess.parse_sess,
+                                           sym::raw_dylib,
+                                           span.unwrap_or_else(|| syntax_pos::DUMMY_SP),
+                                           GateIssue::Language,
+                                           "kind=\"raw-dylib\" is feature gated");
```

Comment:
```suggestion
                                           "kind=\"raw-dylib\" is experimental");
```

Reply:
The error text is copied from one line above, the "static-nobundle" kind. Could we defer this change to a later PR? I think we can update these texts and their unit tests together.

Reply:
I'll leave that up to @nagisa.

Reply:
Changed to "unstable" per #64263.